modules/light-clients/08-wasm/internal/globals/globals.go

```diff
@@ -0,0 +1,34 @@
+package globals
```

Comment:
Open to better names.

Reply:
We could just `wasm`? And then rename `wasmVM` to `vm`, as it would stutter otherwise

Reply:
I'd be happy with `08-wasm/internal/globals.go` and import as `internal.WasmVM` or `internal.GetVM()`, either is fine by me. Don't really know if we need to create a whole directory for the globals.. but I'm okay with whatever y'all decide

Reply:
isn't it convention to have a file named wasm that corresponds and adds funcs for the proto with a similar name?

I'm fine with globals. It can chill there and be a permanent reminder of how horrible globals are.

Reply:
I think as long as the import in the go code is not `globals` I'm fine with a globals.go file. It's not really global sinne the scope is limited to the wasm directory.

Reply:
> Don't really know if we need to create a whole directory for the globals.. but I'm okay with whatever y'all decide

I tried to do it without the subdirectory, but then the package was not internal and I was still able to import it from outside of the module...